submissions/onyinyeChukwuka/HTML/index.html

```diff
@@ -1,71 +1,117 @@
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product Page</title>
+    <link href="../css/productpage.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Raleway&display=swap" rel="stylesheet">
 </head>
+
 <body>
     <main>
-        <header>
-            <p> <h1> BOOTSTRAPS </h1></p>
+        <!-- FOR THE HEADER -->
+        <div class="header">
             <nav>
-                <input type="text" placeholder="Search">
-                <input type="Submit" value="Submit">
-                <ul>
-                    <li> <a href="#"> Home </a></li>
-                    <li> <a href="#"> WOMEN</a></li>
-                    <li> <a href="#"> MEN  </a></li>
-                    <li> <a href="#"> Hot Sales </a></li>
-                    <li> <a href="../html-forms/contactform.html" target="_blank"> Contact Us </a></li>
-                    <li> <a href="#"> Sign in </a></li>
-                    <li> <a href="#"> Cart </a></li>
-                </ul>
+                <div class="left-side">
+                    <div class="tab">
+                        <strong>BOOTSTRAPS</strong>
+                    </div>
+                </div>
+                <div class="right-side">
+                    <div class="link-wrapper">
+                        <a href="../html/index.html"> Product Page </a>
+                    </div>
+                    <div class="link-wrapper">
+                        <a href="../html-forms/contactform.html">Contact Us</a>
+                    </div>
+                    <div class="link-wrapper">
+                        <a href="../css/sign-up.html">Sign-up</a>
+                    </div>
+                </div>
             </nav>
-            <hr>
-        </header>
+        </div>
+        <div class="display">
+            <!-- FOR THE LEFT SIDE -->
+            <div class="cami">
+                <img src="images/combatboots4.jpg" width="469px" height="254px">
+            </div>
 
-        <p> <h2> Cami's Combat Boots</h2>
-            <img src="images/combatboots1.jpg">
-            <img src="images/combatboots4.jpg">
-            <br> 
-            <br>
-            <hr>
-            <hr>
-            <br>
-            <strong> Price:</strong> &#x20A6; 25,000
-            <h4> Cami's Combat Boots</h4>
-            <hr>
-            <strong> Features</strong>
-            <br>
-            <ul>
-                <li> Round toe </li>
-                <li> Padded cuff  </li>
-                <li> Chunky sole</li>
-                <li> Lace-up fastening</li>
-            </ul>
-        </p>
-        <p>
-            <strong> DESCRIPTION</strong>
-            <br>
-            - Heavy laced leather boot especially with a wide cuff buckled above the ankle. Suitable for outdoor activities and casual outings.
-        </p>
+            <!-- FOR THE RIGHT SIDE -->
```

Comment:
When another developer is going through your code and they don't have your UI, this comment might be a lil' bit confusing.